content/en/Getting started/aws.md

```diff
@@ -79,6 +79,9 @@ name: hello
 environments:
   - name: staging
     path: "opta.yaml" # the file we created in previous step
+input_variables:
+  - name: public_uri
+    default: "/hello"
```

Comment:
I wonder if we should do this - I think the getting started should be as minimal as possible.
Changing the uri end point is not bringing any significant value when just getting started.

Reply:
This just shows users some basic capabilities, and I want to make sure all are aware of our templatizations

Reply:
Here is the terraform getting started
https://learn.hashicorp.com/tutorials/terraform/aws-build?in=terraform/aws-get-started

they have a similar feature for input variables but their getting started is very lean and it's not mentioning it. I think we should do the same.

getting started goal is to cover the key feature of opta - this is more advanced.

Reply:
we can add it in Next steps though

Reply:
@NitinAgg what do you think we should do with this one?